docs/en/operations/server_settings/settings.md

```diff
@@ -707,4 +707,32 @@ For more information, see the section "[Replication](../../operations/table_engi
 </zookeeper>
 ```
 
+## use_minimalistic_part_header_in_zookeeper {#server-settings-use_minimalistic_part_header_in_zookeeper}
+
+Storage method of the data parts headers in ZooKeeper.
+
+This setting applies only to the `MergeTree`-family and it can be specified:
+
+- Globally in the `config.xml`.
+
+    ClickHouse use the setting for all the tables on the server. You can change the setting at any time. Existing tables change their behavior with the setting change.
+
+- For each separate table.
+
+    When creating a table, specify the corresponding [engine setting](../table_engines/mergetree.md#table_engine-mergetree-creating-a-table). Ones creating a table you can't change its behavior even with the global setting.
```

Comment:
After creating...

Reply:
I think "When creating"  because `create table ... engine MergeTree() settings use_minimalistic_part_header_in_zookeeper = 1`

Reply:
It wasn't clear, but I've meant the second sentence, instead of "Ones creating"